deploy/lvm-operator.yaml

```diff
@@ -1227,7 +1229,7 @@ spec:
             - "--leader-election=true"
           imagePullPolicy: IfNotPresent
         - name: csi-provisioner
-          image: k8s.gcr.io/sig-storage/csi-provisioner:v2.1.0
+          image: k8s.gcr.io/sig-storage/csi-provisioner:v3.0.0
```

Comment:
We are keeping the Snapshotter version as same as earlier , since the latest version only support the V1 snaphot apis , and cloud k8s distribution, openshift etc does not supported the v1 snapshot APIs yet.